mne/viz/_3d.py

```diff
@@ -1184,11 +1184,15 @@ def _make_tris_fan(n_vert):
 
 def _sensor_shape(coil):
     """Get the sensor shape vertices."""
-    rrs = np.empty([0, 2])
-    tris = np.empty([0, 3], int)
+    from scipy.spatial import ConvexHull
```

Comment:
what is the convention in placing imports inside the functions?

Reply:
Some scipy imports are nested to improve `import mne` speed. There is a test for violations in `mne/tests/test_import_nesting.py::test_module_nesting` and the rules are listed there